common/models/user.js

```diff
@@ -7,6 +7,41 @@ var config = require('../../server/config.json');
 var path = require('path');
 
 module.exports = function(User) {
+
+//send email again incase of unverified email
+User.triggerEmail = function(userId, cb) {
+  User.find({where: {id: userId}}, function(err, user)
+  {
+    const userObj = user[0];
+      var options = {
+      type: 'email',
+      to: userObj.email,
+      from: 'noreply@loopback.com',
+      subject: 'Thanks for registering.',
+      template: path.resolve(__dirname, '../../server/views/verify.ejs'),
+      redirect: '/verified',
+      user: userObj
+    };
+    userObj.verify(options, function(err, response) {
+      if (err) {
+        User.deleteById(userObj.id);
```

Comment:
@bajtos PTAL
If login fails, I am giving user an option to trigger an email incase his token has expired or try logging in again incase he entered bad credentials.

<img width="877" alt="screen shot 2017-05-25 at 1 38 51 pm" src="https://cloud.githubusercontent.com/assets/15273582/26469626/8724d03c-414f-11e7-9ea3-70156a3a8c00.png">